rmc-docs/src/limitations.md

```diff
@@ -0,0 +1,401 @@
+10.1.13 | Pointer types | Partial | See [Observations - Advanced features](#advanced-features) |
+10.1.14 | Function pointer types | Partial | See [Observations - Advanced features](#advanced-features) |
+10.1.15 | Trait object types | Partial | See [Observations - Advanced features](#advanced-features) |
+10.1.16 | Impl trait type | Partial | See [Observations - Advanced features](#advanced-features) |
+10.1.17 | Type parameters | Partial | See [Observations - Advanced features](#advanced-features) |
+10.1.18 | Inferred type | Partial | See [Observations - Advanced features](#advanced-features) |
+10.2 | Dynamically Sized Types | Partial | See [Observations - Advanced features](#advanced-features) |
+10.3 | Type layout | Yes | |
+10.4 | Interior mutability | Yes | |
+10.5 | Subtyping and Variance | Yes | |
+10.6 | Trait and lifetime bounds | Yes | |
+10.7 | Type coercions | Partial | See [Observations - Advanced features](#advanced-features) |
+10.8 | Destructors | Partial | |
+10.9 | Lifetime elision | Yes | |
+11 | Special types and traits | Partial | |
+| | `Box<T>` | Yes | |
+| | `Rc<T>` | Yes | |
+| | `Arc<T>` | Yes | |
+| | `Pin<T>` | Yes | |
+| | `UnsafeCell<T>` | Partial | |
+| | `PhantomData<T>` | Partial | *Review this* |
+| | Operator Traits | Partial | |
+| | `Deref` and `DerefMut` | Yes | |
+| | `Drop` | Partial | |
+| | `Copy` | Yes | |
+| | `Clone` | Yes | |
+14 | Linkage | Yes | |
+15.1 | Unsafe functions | Yes | |
+15.2 | Unsafe blocks | Yes | |
+15.3 | Behavior considered undefined | Partial | |
+| | Data races | No | See [Observations - Concurrency](#concurrency) |
+| | Dereferencing dangling raw pointers | Yes | |
+| | Dereferencing unaligned raw pointers | No | |
+| | Breaking pointer aliasing rules | No | |
+| | Mutating immutable data | No | |
+| | Invoking undefined behavior via compiler intrinsics | Partial | See [Observations - Intrinsics](#intrinsics) |
+| | Executing code compiled with platform features that the current platform does not support | No | |
+| | Producing an invalid value, even in private fields and locals | No | |
+
+## Observations on partially or unsupported features
+
+### Code generation for unsupported features
+
+RMC aims to be a industrial verification tool. Most industrial crates may
+include unsupported features in parts of their code that do not need to be
+verified. In general, this should not prevent users from verifying their code.
+
+Because of that, the general rule is that RMC generates an `assert(false)`
+statement when compiling any unsupported feature. This will cause proofs to fail
+if the statement is reachable during the verification stage. However, the
+analysis will not affected if the statement is not reachable from the code under
+verification, so users can still verify components of their code not using
+unsupported features.
+
+### Assembly
+
+CBMC support for assembly code is quite limited. It may be extended in the
```

Comment:
I would say RMC instead of CBMC here. At this point, I would only mention CBMC if you want to add some link to CBMC assembly support.